crates/net/network/src/transactions.rs

```diff
@@ -312,6 +312,12 @@ where
                     msg,
                 })
             }
+            NetworkEvent::PeerAdded(_) => {
+                todo!()
+            }
+            NetworkEvent::PeerRemoved(_) => {
+                todo!()
+            }
```

Comment:
we don't want to panic here, just ignore.

```suggestion
            _ => {}
```